src/index.html

```diff
@@ -4,9 +4,394 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="qwert"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+          href="#home"
+          class="logo"
+      >
+        <img
+            src="../src/images/logo.png"
+            alt="MOYO-logo"
+            class="moyo-logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+                href="#apple"
+                class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+                href="#samsung"
+                class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+                href="#smartphones"
+                class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+                href="#laptops"
+
+                class="nav__link"
+
+                data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+                href="#gadgets"
+                class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+                href="#tablets"
+                class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+                href="#photo"
+                class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+                href="#video"
+                class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+
+    </header>
+
+    <main class="cards-container">
+      <section class="product-cards">
+        <div class="card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27' Retina 5K Monoblock (MNED2UA/A)"
+            class="card__img"
+          >
+          <h3 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
```

Comment:
```suggestion
          >
          
          <h3 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
```

Reply:
Fix everywhere
<img width="1058" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/86789161/d28026cf-de82-43bb-ac94-a6b07e1b1105">

Reply:
sorry, but I don't understand clear my mistakes(

Reply:
Dmytro had in mind to add a blank line between sibling elements